File: Diplom/Controllers/Course/Contract/CourseDto.cs
namespace Diplom.Controllers.Course.Contract;

public class CourseDto
{
    public int Id { get; set; }
    public int SubjectId { get; set; }
    public string SubjectName { get; set; } = "";
    public int LectorId { get; set; }
    public string LectorFullName { get; set; } = "";
    public DateTime StartDate { get; set; }
    public DateTime EndDate { get; set; }
}

File: DiplomMobile/FaceAttend/app/src/main/java/com/example/faceattend/api/response/CourseDto.kt
package com.example.faceattend.api.response

data class CourseDto(
    val id: Int,
    val subjectId: Int,
    val subjectName: String,
    val lectorId: Int,
    val lectorFullName: String,
    val startDate: String,
    val endDate: String
)